core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -457,33 +463,51 @@ public static AlluxioConfiguration merge(AlluxioConfiguration conf, Map<?, ?> pr
   }
 
   /**
-   * Loads cluster default values from the meta master.
+   * Sends a request to meta master to set a property for a path pattern.
    *
-   * @param address the master address
-   * @param conf configuration to use
-   * @return a configuration object containing the original configuration merged with cluster
-   *         defaults, or the original object if the cluster defaults have already been loaded
+   * @param address the meta master address
+   * @param conf the cluster level configuration
+   * @param path the path pattern
+   * @param key the property key
+   * @param value the property value
    */
-  public static AlluxioConfiguration loadClusterDefaults(InetSocketAddress address,
-      AlluxioConfiguration conf)
-      throws AlluxioStatusException {
-    if (!conf.getBoolean(PropertyKey.USER_CONF_CLUSTER_DEFAULT_ENABLED)
-        || conf.clusterDefaultsLoaded()) {
-      return conf;
+  public static void setPathConfiguration(InetSocketAddress address, AlluxioConfiguration conf,
+      String path, PropertyKey key, String value) throws AlluxioStatusException {
+    GrpcChannel channel = null;
+    try {
+      channel = GrpcChannelBuilder.newBuilder(address, conf).disableAuthentication().build();
+      MetaMasterConfigurationServiceGrpc.MetaMasterConfigurationServiceBlockingStub client =
+          MetaMasterConfigurationServiceGrpc.newBlockingStub(channel);
+      client.setPathConfiguration(SetPathConfigurationPRequest.newBuilder()
+          .setPath(path).setKey(key.getName()).setValue(value).build());
+    } catch (io.grpc.StatusRuntimeException e) {
```

Comment:
Is there a class conflict?

Reply:
Moved this method to `MetaMasterConfigClient`